packages/office-ui-fabric-react/src/components/SwatchColorPicker/SwatchColorPicker.types.ts

```diff
@@ -74,7 +74,10 @@ export interface ISwatchColorPickerProps {
    * If `id` and `color` are unspecified, cells are no longer being focused.
    */
   onCellFocused?: (id?: string, color?: string) => void;
-
+  /**
+   * Custom render function for the color cell
+   */
+  onRenderColorCell?: IRenderFunction<IColorCellProps>;
```

Comment:
Do we have any UT for SwatchColorPicker ? If yes, then can we add one UT for with this new prop ?

Reply:
Yes we have a UT for SwatchColorPicker and I have added a scenario for it.